pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTest.java

```diff
@@ -209,7 +210,7 @@ void internalConfiguration() throws Exception {
         InternalConfigurationData expectedData = new InternalConfigurationData(
```

Comment:
Does this have any credential information?

Reply:
its does not contain the keys